lib/index.js

```diff
@@ -577,6 +577,20 @@ class RustLanguageClient extends AutoLanguageClient {
       throw new Error("failed to start server: " + e)
     }
   }
+
+  provideOutlines() {
```

Comment:
I think we should add a comment on this method just briefly explaining what we're doing & the motivation for filtering variables.